webui/react/src/hooks/useModal/Experiment/useModalExperimentCreate.tsx

```diff
@@ -162,20 +163,20 @@ const useModalExperimentCreate = ({ onClose }: Props = {}): ModalHooks => {
         setModalState((prev) => {
           if (!prev) return prev;
 
-          if (prev.isAdvancedMode && formRef.current) {
+          if (prev.isAdvancedMode && form) {
             try {
               const newConfig = (yaml.load(prev.configString) || {}) as RawJson;
               const isFork = prev.type === CreateExperimentType.Fork;
 
-              formRef.current.setFields([
+              form.setFields([
                 { name: 'name', value: getExperimentName(newConfig) },
                 {
                   name: 'maxLength',
                   value: !isFork ? getMaxLengthValue(newConfig) : undefined,
                 },
               ]);
 
-              formRef.current.validateFields();
+              form.validateFields();
```

Comment:
i think `form.validateFields();` should be `await form.validateFields();` 
https://ant.design/components/form?theme=dark#forminstance